plugins/systemd/systemd_mem

```diff
@@ -51,28 +58,51 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SO
 
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
+# at least on debian stretch services report this as unlimited / unknown value
+SYSTEMD_UNLIMITED="18446744073709551615"
```

Comment:
This value is `2 ^ 64`, thus it may be suitable only for 64 bit machines?
Anyway: maybe there is no alternative for using this hard-coded number. Thus a reference to the 64 bit issue in the comment should be sufficient.